src/config.rs

```diff
@@ -192,12 +192,24 @@ const KDF_ROUNDS: &str = "R";
 pub enum KdfConfig {
     /// Derive keys with repeated AES encryption
     Aes { rounds: u64 },
-    /// Derive keys with Argon2
+    /// Derive keys with Argon2d
     Argon2 {
```

Comment:
If this is actually the `2d` variant, should we not rename the struct as well?
Doesn't have to happen in this PR, it could happen before `1.x` since the API is changing quite a lot at the moment.

Reply:
I definitely can do it. 

If we break the API, should i add support for Aigon2i as well? (In two seperate PRs?)

Reply:
> If we break the API, should i add support for Aigon2i as well?

If it's offered in applications that use KDBX, then I feel like we will get the request for it at some point. Might as well break the API before we reach `1.0`.

> In two seperate PRs?

Yes please, this one is ready to merge.

Reply:
@tlercher FYI created https://github.com/sseemayer/keepass-rs/issues/166 to track the `Argon2i` feature work.